Source/core/ProcessInfo.cpp

```diff
@@ -32,6 +32,7 @@
 #include <unistd.h>
 #endif
 
+#include "tracing/TraceCategories.h"
```

Comment:
This is not possible. This give cyclyc dependencies. You can not use tracing in core (only TRACE__LX allowed here)